addons/point_of_sale/static/src/app/navbar/closing_popup/closing_popup.js

```diff
@@ -146,7 +140,7 @@ export class ClosePosPopup extends Component {
         return true;
     }
     async closeSession() {
-        this.customerDisplay?.update({ closeUI: true });
+        // TODO: show pos is closed sign on customer display
```

Comment:
make sure that when the session is closed the iot box goes back to the default view